lib/pure/nativesockets.nim

```diff
@@ -482,8 +483,18 @@ proc getAddrString*(sockAddr: ptr SockAddr): string =
         raiseOSError(osLastError())
     setLen(result, len(cstring(result)))
   else:
+    when defined(posix) and not defined(nimdoc):
+      if sockAddr.sa_family.cint == nativeAfUnix:
+        return "unix"
     raise newException(IOError, "Unknown socket family in getAddrString")
 
+when defined(posix) and not defined(nimdoc):
```

Comment:
This is bad because you won't see the procedure in the documentation. Can you also do what you did in the other file?

Reply:
true